tests/server/jobs/test_endpoint.py

```diff
@@ -29,8 +31,42 @@ def job_assert_tracking_uri(server_url: str) -> None:
     assert mlflow.get_tracking_uri() == server_url
 
 
+class Client:
+    def __init__(self, server_url: str):
+        self.server_url = server_url
+
+    def submit_job(
+        self, function_fullname: str, params: dict[str, Any], timeout: float | None = None
+    ) -> str:
```

Comment:
```suggestion
    ) -> Job:
```

can we return a job object?

Reply:
I suggest to return a json dict instead.
the endpoint is for UI invocation, and UI side will receive the json data. so that we should verify the json data in tests.